node/resolvers/createReturnRequest.ts

```diff
@@ -169,7 +172,49 @@ export const createReturnRequest = async (
     ],
   })
 
-  // TODO: Send confirmation email.
+  try {
```

Comment:
Nice! I think it's having the try catch here is a nice addition (we usually don't need it inside a resolver). I would only add a comment saying that we are doing it here to avoid sending an error to the browser when only the email fails in the process.